### ISSUE TITLE
@wordpress/eslint-plugin 9.3.0 causes errors when running "wp-scripts lint-js"

### ISSUE THREAD

**Issue by peroks** [DekodeInteraktiv/project-base#330]
The current version of `@wordpress/eslint-plugin` (v9.3.0) doesn't play nicely in **Project base**.

After upgrading from @wordpress/eslint-plugin 9.2.0 to 9.3.0, I get about 50 of the below errors when I run `wp-scripts lint-js`

```
/Users/dekode/Sites/mills/app/.eslintrc.js
  0:0  error  Parsing error: No Babel config file detected for /Users/dekode/Sites/mills/app/.eslintrc.js. Either disable config file checking with requireConfigFile: false, or configure Babel so that it can find the config files

/Users/dekode/Sites/mills/app/packages/plugins/mills-block-contact/src/edit.js
  0:0  error  Parsing error: No Babel config file detected for /Users/dekode/Sites/mills/app/packages/plugins/mills-block-contact/src/edit.js. Either disable config file checking with requireConfigFile: false, or configure Babel so that it can find the config files
```

In .eslintrc.js, after setting

```
parserOptions: {
   requireConfigFile: false,
}
```

I get even more errors, like

```
Error while parsing /Users/dekode/Sites/mills/app/packages/plugins/mills-block-contact/src/edit.js
Line 13, column 2: This experimental syntax requires enabling one of the following parser plugin(s): 'jsx, flow, typescript' (13:2)
`parseForESLint` from parser `/Users/dekode/Sites/mills/app/node_modules/@babel/eslint-parser/lib/index.cjs` is invalid and will just be ignored

Error while parsing /Users/dekode/Sites/mills/app/packages/plugins/mills-block-featured/src/edit.js
Line 110, column 1: This experimental syntax requires enabling one of the following parser plugin(s): 'jsx, flow, typescript' (110:1)
`parseForESLint` from parser `/Users/dekode/Sites/mills/app/node_modules/@babel/eslint-parser/lib/index.cjs` is invalid and will just be ignored
```

**Comment by peroks** [comment]
@fnpen solved this issue in our project by adding this to `.eslintrc.js`

```
	parser: '@babel/eslint-parser',
	parserOptions: {
		babelOptions: {
			presets: ['@wordpress/babel-preset-default'],
		},
		requireConfigFile: false,
	},

```

**Comment by walbo** [comment]
Closed by #331
